src/cache/DbQueryResultCache.ts

```diff
@@ -60,6 +59,11 @@ export class DbQueryResultCache implements QueryResultCache {
                 type: driver.normalizeType({ type: driver.mappedDataTypes.cacheIdentifier }),
                 isNullable: true
             }),
+            new TableColumn({
```

Comment:
it shouldnt be a separate column i think. Just prepend to cacheIdentifier

Reply:
how? each database will have its own cache table

Reply:
If that is the case then there isn't an actual need to have a prefix for the database cache, it is a redis only requirement.

Reply:
correct, but this option is exist and it global for all caches, and if user uses a table cache and set it and it won't affect anything it will be strange, right?

Reply:
@pleerock So perhaps we should rename the option to make it explicit that it is a redis only option, `redisKeyPrefix` or similar? Doesn't seem like useful to implement it for the database cache if it has zero benefit.

Reply:
no, its a bad idea. Cache can be anything, not only table or redis and its abstract from a real implementation. So probably its better to append prefix to queries and identifier I think. Those who wont use it simply will ignore it.

Reply:
Okay if that's what you want for the database cache implementation I will do that. 

Even as we're leaving it as an abstract option, it does seems arbitrary to implement it in the database caching by prepending to `identifier` (which might be empty) and have zero affect on the end functionality - having it stored in a separate column for me is more explicit and less arbitrary (and more expected) but I figure thats a breaking change for existing users of the database cache (so not great).

Thanks for your time, it's great to have these discussions! I don't have time tonight but I should be able to get that done for tomorrow :)